Dockerfile

```diff
@@ -57,9 +60,11 @@ RUN set -ex; \
     # Clean up
     apk del .build-deps; \
     rm -rf /tmp/pear /var/cache/apk/*
+# DO NOT FORGET TO CHECK THE LANGUAGE PACK DOWNLOAD URL BELOW
 # DO NOT FORGET TO UPDATE "image-version" FILE
-ENV OSTICKET_VERSION=1.15.4 \
-    OSTICKET_SHA256SUM=6c7cf5fe74258dd0beac6176c438468f468a0a3f07f1f6a0349cc68a9e514302
+ENV OSTICKET_VERSION=1.16.1 \
+    OSTICKET_SHA256SUM=4cfb6a297b48f551b0988a7df72448fe7ec22ee38e4023fafc19ead41fb76b38
+COPY osticket-glob-brace.patch /osticket-glob-brace.patch
```

Comment:
This creates a new, useless layer, since the file is unlinked in the next layer. Have you thought about uploading the patch to Gist or something like that? Multi-staged build would be another option, but that sounds like an overkill here.

Reply:
I see no way to create a gist under organization account - so it would have to be on my personal profile. Is that OK with you? An alternative could be to download the file from this repository from `https://raw.githubusercontent.com/devinsolutions/docker-osticket/<commit-hash>/osticket-glob-brace.patch`. However updating that patch would require two commits in that case, and one would need to be careful to e.g. not squash commits when merging PR that contains update of the patch.

Reply:
Digging into this further, another option is to require BuildKit and use `RUN --mount`:

```Dockerfile
# syntax=docker/dockerfile:1.3

[...]

RUN --mount=type=bind,target=/tmp/build-context set -ex; \
[...]
    patch --directory="/var/www/html" --forward --strip=1 \
        --input="/tmp/build-context/osticket-glob-brace.patch"; \

[...]
```

Reply:
Actually, it would appear that the layer cache is invalidated as one would expect, if the patch file changes. I've confirmed it with my local testing, and [this](https://github.com/moby/buildkit/issues/1903#issuecomment-746903174) comment would seem to agree. So I don't think cache-busting will be necessary.